Docs/source/io.rst

```diff
@@ -304,6 +304,8 @@ Derived variables
 +-----------------------------------+---------------------------------------------------+-----------------------------+-----------------------------------------+
 | variable name                     | description                                       | derive routine              | units                                   |
 +===================================+===================================================+=============================+=========================================+
+| ``abar``                          | Mean atomic mass                                  | ``derabar``                 | :math:`\amu`                            |
```

Comment:
we usually write this as `\bar{A}`, but it is equivalent to the mean molecular weight, `\mu`.